doc/glossary.rst

```diff
@@ -14,186 +14,193 @@ reflect either technical terms or legacy ways of referring to Ceph systems.
+                directly on the Ceph block devices without any file system
+                interface.
 
 	Ceph
-	Ceph Platform
-		All Ceph software, which includes any piece of code hosted at
-		`https://github.com/ceph`_.
-
-	Ceph System
-	Ceph Stack
-		A collection of two or more components of Ceph.
+	Ceph Block Device
+	Ceph Block Storage
+                The block storage "product," service or capabilities when used
+                in conjunction with ``librbd``, a hypervisor such as QEMU or
+                Xen, and a hypervisor abstraction layer such as ``libvirt``.
 
-	Ceph Node
-	Node
-	Host
-		Any single machine or server in a Ceph System.
+	Ceph Client
+                The collection of Ceph components which can access a Ceph
+                Storage Cluster. These include the Ceph Object Gateway, the
+                Ceph Block Device, the Ceph File System, and their
+                corresponding libraries, kernel modules, and FUSEs.
 
-	Ceph Storage Cluster
-	Ceph Object Store
-	RADOS
-	RADOS Cluster
-	Reliable Autonomic Distributed Object Store
-		The core set of storage software which stores the user's data (MON+OSD).
+	Ceph Client Libraries
+                The collection of libraries that can be used to interact with
+                components of the Ceph System.
 
+	Ceph Clients
 	Ceph Cluster Map
-	Cluster Map
-		The set of maps comprising the monitor map, OSD map, PG map, MDS map and
-		CRUSH map. See `Cluster Map`_ for details.
-
-	Ceph Object Storage
-		The object storage "product", service or capabilities, which consists
-		essentially of a Ceph Storage Cluster and a Ceph Object Gateway.
-
-	Ceph Object Gateway
-	RADOS Gateway
-	RGW
-		The S3/Swift gateway component of Ceph.
-
-	Ceph Block Device
-	RBD
-		The block storage component of Ceph.
-
-	Ceph Block Storage
-		The block storage "product," service or capabilities when used in
-		conjunction with ``librbd``, a hypervisor such as QEMU or Xen, and a
-		hypervisor abstraction layer such as ``libvirt``.
-
+	Ceph Dashboard
 	Ceph File System
 	CephFS
 	Ceph FS
-		The POSIX filesystem components of Ceph. Refer
-		:ref:`CephFS Architecture <arch-cephfs>` and :ref:`ceph-file-system` for
-		more details.
+                The POSIX filesystem components of Ceph. Refer :ref:`CephFS
+                Architecture <arch-cephfs>` and :ref:`ceph-file-system` for
+                more details.
 
-	Cloud Platforms
-	Cloud Stacks
-		Third party cloud provisioning platforms such as OpenStack, CloudStack,
-		OpenNebula, Proxmox VE, etc.
+	Ceph Interim Release
+                Versions of Ceph that have not yet been put through quality
+                assurance testing, but may contain new features.
 
-	Object Storage Device
-	OSD
-		A physical or logical storage unit (*e.g.*, LUN).
-		Sometimes, Ceph users use the
-		term "OSD" to refer to :term:`Ceph OSD Daemon`, though the
-		proper term is "Ceph OSD".
+	Ceph Test Framework
```

Comment:
@zdover23 This appears to be in the wrong place.

Reply:
Thanks. I put it in the right place.